Aztec/Classes/TextKit/ParagraphProperty/HTMLLi.swift

```diff
@@ -15,6 +15,14 @@ class HTMLLi: ParagraphProperty {
     }
 
     static func ==(lhs: HTMLLi, rhs: HTMLLi) -> Bool {
-        return lhs === rhs
+        return false
```

Comment:
Should this always return false? shouldn't we delegate to the method isEqual ?

Reply:
This should ideally be `lhs == rhs`, but the problem we have is that Aztec (the editor logic) is using the same `HTMLLi` instance for separate list items, so you never get two bullets.  This is not new, it's always been like that but with the previous merging logic it wasn't noticeable.

Making Aztec's logic smart enough to know where to apply `HTMLLi` instances means changes to the list formatter, which is way beyond my current scope with this PR.

Reply:
Ok them, probably flag this on a new issue for us to fix later on.